src/sys/epoll.rs

```diff
@@ -8,12 +8,20 @@ mod ffi {
 
     extern {
         pub fn epoll_create(size: c_int) -> c_int;
+        pub fn epoll_create1(flags: c_int) -> c_int;
         pub fn epoll_ctl(epfd: c_int, op: c_int, fd: c_int, event: *const EpollEvent) -> c_int;
         pub fn epoll_wait(epfd: c_int, events: *mut EpollEvent, max_events: c_int, timeout: c_int) -> c_int;
```

Comment:
Hi @kubo39 , although this is not directly related to your PR we have been moving to defining ffi constants entirely in libc and using those rather than defining our own (see [CONVENTIONS.md](https://github.com/nix-rust/nix/blob/master/CONVENTIONS.md#libc-constants-functions-and-structs).  it looks like the existing constants are present but `epoll_create1` is not.  Can you please do the following:

1. Open up a PR to libc adding `epoll_create1`
2. Update `epoll.rs` to use the ffi definitions from libc.

Also, thanks for the change!